examples/examples.json

```diff
@@ -16,11 +16,6 @@
                     "desc": "Add multiple CARTO layers to your map",
                     "file": "basics/multi-layer.html"
                 },
-                {  
-                    "title": "Change the source",
```

Comment:
Why do you remove two examples?

Reply:
Because `setSource` method was removed

Reply:
Yes, but, the functionality is still there, we should use `update()` instead